src/main.rs

```diff
@@ -57,6 +63,7 @@ fn main() {
         &args.stdout_prefix,
         &args.stderr_prefix,
         &args.exit_status_prefix,
+        args.overwrite || std::env::var("GOLDENTEST_OVERWRITE").is_ok(),
```

Comment:
I'm a bit hesitant on enabling this automatically if an environment variable is defined. It seems unnecessary to me. Do you have an existing use case for this or can we remove it?

Reply:
I added it more like an afterthought. I'm completely fine with removing it.